### PR TITLE
Fix style map key not found

### PR DESCRIPTION
**Pull request by jackyho112**
Add support for missing inline style definition.

### CODE REVIEW

**Comment by MikeKlemarewski** [comment]
What does this fix? Does this just ensure the exporter doesn't blow up if it runs into an inline style it doesn't understand?

File: lib/draftjs_exporter/style_state.rb [hunk 1]
```diff
@@ -18,20 +18,24 @@ def apply(command)
...
-        "#{hyphenize(key)}: #{value};"
-      }.join
+      available_styles
+        .map { |style| style_map.fetch(style) }
+        .inject({}, :merge).map { |key, value|
+          "#{hyphenize(key)}: #{value};"
+        }.join
+    end
+
+    def available_styles
```
Comment (MikeKlemarewski) [inline lib/draftjs_exporter/style_state.rb]: maybe `supported_styles`?

**Comment by jackyho112** [comment]
@MikeKlemarewski Yup

File: Gemfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+gem 'byebug'
```
Comment (jvaill) [inline Gemfile]: This should be added to the `:development` and `:test` groups only so that we don't install this in production. 🙀

**Comment by MikeKlemarewski** [comment]
Looks reasonable to me 👍

**Pull request merged**